linalg/operator.hpp

```diff
@@ -442,6 +442,48 @@ class TimeDependentOperator : public Operator
    virtual ~TimeDependentOperator() { }
 };
 
+// ---------------------------------------------------------------------------
+// TimeDependentAdjointOperator is a TimeDependentOperator with Adjoint rate equations to be used with CVODESSolver
+// ---------------------------------------------------------------------------
+
```

Comment:
I would suggest a longer discussion on the general strategy to be described in the class description. Some references would be helpful also.